XRTK-Core/Packages/com.xrtk.core/Editor/Profiles/InputSystem/Controllers/BaseMixedRealityControllerDataProviderProfileInspector.cs

```diff
@@ -213,8 +213,6 @@ public override void OnInspectorGUI()
                         break;
                 }
             }
-
-            EditorGUILayout.Space();
```

Comment:
@FejZa what does this fix exatly?

Reply:
Doesn't fix anything. It removes a space that is not needed so things in the inspector look cleaner.